apps/src/javalab/CommitDialog.jsx

```diff
@@ -41,24 +56,40 @@ export default class CommitDialog extends React.Component {
       >
         {compileStatusContent}
       </div>,
-      <div key="buttons">{buttons}</div>
+      <div key="buttons">
+        <FooterButton
+          key="cancel"
+          type="cancel"
+          text="Cancel"
+          onClick={this.props.handleClose}
+        />
+        ,
+        <FooterButton
+          id="confirmationButton"
+          key="confirm"
+          text={commitText}
+          disabled={isDisabled}
+          color="green"
+          onClick={() => {}}
+        />
+      </div>
     ];
   };
 
-  toggleFileToCommit = filename => {
-    let filesToCommit = [...this.state.filesToCommit];
-    const fileIdx = filesToCommit.findIndex(name => name === filename);
+  toggleFileToBackpack = filename => {
+    let filesToBackpack = [...this.state.filesToBackpack];
+    const fileIdx = filesToBackpack.findIndex(name => name === filename);
```

Comment:
nit: this can instead be `filesToBackpack.indexOf(filename)`

Reply:
done- thanks!